app/actions/posts.js

```diff
@@ -84,3 +83,84 @@ export const createPost = (title, description, imageData) => {
     })
   };
 };
+
+
+export const likePost = (postId) => {
+  return (dispatch, getState) => {
+    const currentUser = getState().currentUser
+
+    dispatch({
+      type: 'LOAD_SPINNER'
+    });
+
+    return fetch(URL + 'posts/' + postId + '/like', {
```

Comment:
you can use template strings here `${URL}posts/${postId}/like` and that will clean it up